### DISCUSSION TITLE
Issue with Export template

### DISCUSSION THREAD

**Discussion by SeaGooseBlk** [PrateekKumarSingh/AzViz#82] [Q&A]
"Export-AzResourceGroup : Export template is not supported for resource groups that have more than '200' resources."
I am guessing that there is a solution to this, but I am possibly not seeing it.  Has anyone run into this before?

**Comment by PrateekKumarSingh** [comment]
Hey @SeaGooseBlk, this is a [known issue and limitation](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/export-template-portal#limitations) in exporting the Azure resource group template, I'm tracking this under #21 

There might be ways to get around this, but right now I don't have a solution for this issue.
